addons/escoria-core/game/core-scripts/esc_item.gd

```diff
@@ -120,8 +120,8 @@ export(int) var speed: int = 300
 # Speed damp of this item if movable
 export(float) var v_speed_damp: float = 1.0
 
-# Animations script (for walking, idling...)
-export(Script) var animations
+# ESCAnimationsResource (for walking, idling...)
+export(Resource) var animations
 
```

Comment:
You can actually hint at a custom resource with a little hack by using _get_property_list to define the properties and hint at the resource like here: https://github.com/deep-entertainment/egoventure/blob/8fe70a5e24cfb92c9efc1916cb7930fe96252a48/addons/egoventure/resources/in_game_configuration.gd#L47

This makes it easier to select or create the specific resource.